x/evmutil/keeper/conversion_evm_native_bep3.go

```diff
@@ -0,0 +1,55 @@
+package keeper
+
+import (
+	"math/big"
+
+	errorsmod "cosmossdk.io/errors"
+	"github.com/kava-labs/kava/x/evmutil/types"
+)
+
+var (
+	bep3Denoms = map[string]bool{
```

Comment:
Nice, normally I'd say use a `map[string]struct{}`, but with 4 items the benefited clarify and simpler logic in `isBep3Asset` is nice 👍